.github/files/update-project-mirrors.sh

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash
+
+git_setup()
+{
+  cat <<- EOF > "$HOME/.netrc"
+		machine github.com
+			login matticbot
+			password $GITHUB_TOKEN
+		machine api.github.com
+			login matticbot
+			password $GITHUB_TOKEN
+EOF
+  chmod 600 "$HOME/.netrc"
+
+  git config --global user.email "matticbot@users.noreply.github.com"
+  git config --global user.name "matticbot"
+}
+
+# Halt on error
+set -eo pipefail
+
+if [[ -n "$CI" ]]; then
+	git_setup
+fi
+
+BASE=$(pwd)
+CLONE_BASE=$(mktemp -d "${TMPDIR:-/tmp}/jetpack-update-project-mirrors.XXXXXXXX")
+MONOREPO_COMMIT_MESSAGE=$(git show -s --format=%B $GITHUB_SHA)
+COMMIT_MESSAGE=$( echo "${MONOREPO_COMMIT_MESSAGE}\n\nCommitted via a GitHub action: https://github.com/automattic/jetpack/runs/${GITHUB_RUN_ID}" )
+COMMIT_ORIGINAL_AUTHOR="${GITHUB_ACTOR} <${GITHUB_ACTOR}@users.noreply.github.com>"
+
+if [[ "$GITHUB_REF" =~ ^refs/heads/ ]]; then
+	BRANCH=${GITHUB_REF#refs/heads/}
```

Comment:
With how the action is setup, this would, in effect, make sure that pushes to monorepo `master` go to mirror repo `master` and pushes to `feature/reorg` go to a `feature/reorg` branch on the mirror?

Reply:
Exactly.